extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -496,5 +481,70 @@ private boolean isEqual(State state1, State state2) {
         }
         return colorModeIsEqual && effectIsEqual;
     }
-
+    
+	private boolean isReachable() {
+		try {
+			// note that InetAddress.isReachable is unreliable, see
+			// http://stackoverflow.com/questions/9922543/why-does-inetaddress-isreachable-return-false-when-i-can-ping-the-ip-address
+			// That's why we do an HTTP access instead
+
+			if (bridge != null) {
+				// If there is no connection, this line will fail
+				bridge.authenticate("invalid");
+			}
+		} catch (IOException e) {
+			return false;
+		} catch (ApiException e) {
+			if (e.getMessage().contains("SocketTimeout") || e.getMessage().contains("ConnectException")
+					|| e.getMessage().contains("NoRouteToHostException")) {
+				return false;
+			} else {
+				// this seems to be only an authentication issue
+				return true;
+			}
+		}
+		return true;
+	}
+
+	@Override
+	public Collection<ConfigStatusMessage> getConfigStatus() {
+		logger.debug("IP configuration check started.");
+
+		// Error handler bound to the IP address field
+		ConfigStatusInfo configStatus = new ConfigStatusInfo();
+		ConfigStatusMessage.Builder errBuilder = ConfigStatusMessage.Builder.error(HOST);
+
+		// Bridge IP address to be used for checks
+		final String bridgeIpAddress = (String) getThing().getConfiguration().get(HOST);
+
+		// Check whether an IP address is provided
+		if (bridgeIpAddress == null) {
+			logger.error("No IP address has been provided.");
+			configStatus.add(errBuilder.withMessageKey(HueConfigStatusMessage.CONFIG_STATUS_ERROR_PREFIX
```

Comment:
Why not introducing a method "getMessageKey" for HueConfigStatusMessage?
So we can use HostConfigStatusMessage.IP_ADDRESS_MISSING.getMessageKey() instead of building the prefix + code on every usage.